config/cms.conf.sample

```diff
@@ -151,12 +151,14 @@
 
     "_section": "ScoringService",
 
-    "_help": "List of URLs (with embedded username and password) of the",
+    "_help": "List of [contest id, URL] tuples.",
+    "_help": "Contest id specifies which contest scores to send.",
+    "_help": "URL (with embedded username and password) of the",
     "_help": "RWSs where the scores are to be sent. Don't include the",
     "_help": "load balancing proxy (if any), just the backends. If any",
     "_help": "of them uses HTTPS specify a file with the certificates",
     "_help": "you trust.",
-    "rankings": ["http://usern4me:passw0rd@localhost:8890/"],
+    "rankings": [[1, "http://usern4me:passw0rd@localhost:8890/"]],
```

Comment:
This somewhat breaks the current workflow, because now you have to update `cms.conf` every time your contests change.

I suppose that the _real_ fix would be to make RWS contest-aware, that is, serve multiple results pages under a prefix when multi-contest mode is on, similarly to how CWS works now. (Although this would interfere with RWS's feature of joining "day 1" and "day 2" contests…)

Maybe we could use a default of `0` or `-1` here to indicate that the old behavior should be used, so that users with existing workflows don't need to do anything. I would appreciate opinions on this.

Reply:
Additionally, with change to "rankings" property format in config file, `cms/cmscontrib/RWSHelper.py` retrieving of ranking server urls needs to be fixed.